pkg/virtualization/core/service/dvddrive.go

```diff
@@ -144,3 +123,8 @@ func (vmms *VirtualSystemManagementService) RemoveDvdDrive(dvd *drive.DvdDrive)
 	err = vmms.RemoveVirtualSystemResource(dvd.CIM_ResourceAllocationSettingData, -1)
 	return
 }
+
+func (vmms *VirtualSystemManagementService) RemoveDvdDisk(dvddisk *disk.LogicalDisk) (err error) {
```

Comment:
If you add new method, please add a unit test in _test.go

Reply:
This function is just added as a wrapper to call the logical disk deletion. This is already part of existing unit tests and run as part of function `vmms.RemoveISODisk(isodisk)`
I can change the references from of logical disk removal to this function but there is no value add. Please let me know in case I need to add it.

Reply:
Added unit test as part of function `TestAddRemoveIsoDiskByPath()`